packages/utilities/psammead-styles/src/font-families.js

```diff
@@ -376,7 +376,7 @@ export const sinhala = sinhalaStyles;
 export const somali = helmetFontStyles;
 export const swahili = helmetFontStyles;
 export const tamil = tamilStyles;
-export const telugu = teleguStyles;
+export const telugu = teluguStyles;
```

Comment:
Technically isn't this actually a breaking change, therefore needing a major version change?

Reply:
technically, yes

Reply:
oh wait no it isn't because the export is ACTUALLY spelt correctly!

Reply:
Technically, no. Because the export name did not change. That was always correct

Reply:
ahhhhhh 🙈 on that note, time for the weekend 🏃